var/spack/repos/builtin/packages/taskflow/package.py

```diff
@@ -16,20 +16,24 @@ class Taskflow(CMakePackage):
     git = "https://github.com/taskflow/taskflow.git"
 
     version("master", branch="master")
-    version("2.7.0", sha256="bc2227dcabec86abeba1fee56bb357d9d3c0ef0184f7c2275d7008e8758dfc3e")
```

Comment:
We tend to discourage dropping versions of packages without first deprecating them (see https://spack.readthedocs.io/en/latest/packaging_guide.html#deprecating-old-versions).